### ISSUE TITLE
custom_targets in Mbed Studio 1.4.2, mbed-os 6.13.0

### ISSUE THREAD

**Issue by Fleckz** [JojoS62/custom_targets#3]
Hello, so I cloned custom_targets to the project, as target selected DEVEBOX_H743VI, moved custom_targets.json to root of the project directory - trying to build and getting this error: [ERROR] Parameter name 'lse_drive_load_level' defined in both 'target:DEVEBOX_H743VI' and 'target:MCU_STM32H7'
ideas? Thanks!

**Comment by JojoS62** [comment]
yes, the setting lse_drive_load_level is now included in targets.json for all MCU_STM32H7. It can be removed from the custom_targets.json.
I will update my repo soon.

**Comment by Fleckz** [comment]
Yeah, however, that doesn't help much, I get next error:
[Error] mbed_pinmap_default.cpp@73,9: use of undeclared identifier 'CONSOLE_TX'
[Error] mbed_pinmap_default.cpp@73,21: use of undeclared identifier 'CONSOLE_RX'
[Error] mbed_pinmap_default.cpp@76,15: invalid application of 'sizeof' to an incomplete type 'const PinName []'
[Error] mbed_pinmap_default.cpp@97,53: use of undeclared identifier 'CONSOLE_TX'
[ERROR] ./mbed-os/hal/source/mbed_pinmap_default.cpp:73:9: error: use of undeclared identifier 'CONSOLE_TX'

**Comment by JojoS62** [comment]
its hard to follow the frequent changes in Mbed, I'm trying also to compile with current mbed-os-6.

**Comment by JojoS62** [comment]
I have updated the repo and fixed the CONSOLE_TX/RX and lse_drive_load_level setting.
There is another new setting necessary for the system_power_supply, I have set it to "PWR_LDO_SUPPLY", but it maybe needs "PWR_DIRECT_SMPS_SUPPLY". I don't have this target handy for testing, so please report if this working. I will further check what is needed for this target.

**Comment by Fleckz** [comment]
Compiles and works, thank you!!!

**Comment by jeromecoutant** [comment]
Hi

Just for information, there is now a new repo to share all custom boards based on stm32 :-)
https://github.com/ARMmbed/stm32customtargets

You can also be helped by the generic board branch:
https://github.com/ARMmbed/stm32customtargets/tree/GENERIC_TARGET/TARGET_STM32H7

Regards,
jerome

**Comment by Fleckz** [comment]
Thank you a lot! Will check it out!

**Comment by JojoS62** [comment]
Yes, Jerome‘s Repo is also a valuable resource. I need to add more configuration for cmake for other targets.
Some of my targets are really custom, the popular china boards could be included also in Jerome’s repo as a common source.
